classes/template.php

```diff
@@ -255,17 +255,15 @@ public function delete_element($elementid) {
      */
     public function generate_pdf(bool $preview = false, int $userid = null, bool $return = false) {
         global $CFG, $DB, $USER;
-	defined('MOODLE_INTERNAL') || die();
-
+	
 	if (empty($userid)) {
             $user = $USER;
         } else {
             $user = \core_user::get_user($userid);
         }
 
         require_once($CFG->libdir . '/pdflib.php');
-	
-        // Get the pages for the template, there should always be at least one page for each template.
+	// Get the pages for the template, there should always be at least one page for each template.
```

Comment:
Why are you changing the spacing?

Reply:
I didn´t realize, we can leave the original spacing

Reply:
Please fix the spacing.